packages/d3fc-series/src/webgl/bar.js

```diff
@@ -0,0 +1,56 @@
+import xyBase from "../xyBase"
+import { glBar, scaleMapper } from "@d3fc/d3fc-webgl"
+import { exclude, rebind, rebindAll } from "@d3fc/d3fc-rebind"
+
+export default () => {
+    const base = xyBase()
+
+    let context = null
+    let xScale = null
+    let yScale = null
+
+    const draw = glBar()
+
+    const bar = (data) => {
+        const xScale = scaleMapper(base.xScale())
+        const yScale = scaleMapper(base.yScale())
+
+        const x = []
```

Comment:
In line these are `Float32Array`s, I assume this was done to allow them to be passed straight into gl. Are we consciously choosing not to do this here?